domaintools/api.py

```diff
@@ -65,7 +65,11 @@ def _rate_limit(self):
         """Pulls in and enforces the latest rate limits for the specified user"""
         self.limits_set = True
         for product in self.account_information():
-            self.limits[product['id']] = {'interval': timedelta(seconds=60 / float(product['per_minute_limit']))}
+            self.limits[product['id']] = {'interval': timedelta(
+                #  momentarily fix to avoid errors when there is not limit set:
+                seconds=60 / float(product['per_minute_limit'] or 3600))
```

Comment:
@CAMIRO I believe this will take care of the error, but it doesn't handle the per_hour_limit. I'll defer to @mccarthym  here for guidance but think we may need to look for per_minute_limit first, per_hour_limit (doing the timedelta conversion) second, and then default to 3600 as you have here.